Tests/Clappr_Tests/Classes/Plugin/Core/MediaControl/PlayButtonTests.swift

```diff
@@ -194,6 +193,140 @@ class PlayButtonTests: QuickSpec {
                     expect(playButton.view.isHidden).to(beFalse())
                 }
             }
+            
+            describe("#togglePlayPause") {
+                var core: CoreStub!
+                var playButton: PlayButton!
+                
+                beforeEach {
+                    core = CoreStub()
+                    playButton = PlayButton(context: core)
+                    playButton.render()
+                }
+
+                context("when playback state is paused") {
+                    context("can play is true") {
+                        it("calls play") {
+                            core.playbackMock?.state = .paused
+                            core.playbackMock?._canPlay = true
```

Comment:
Now that we moved this conditional into `Container` there is no reason for this arrange here.